sites/friday/src/get-started/stacks/django.md

```diff
@@ -9,21 +9,113 @@ description: |
     It includes common reference materials useful for deploying Django, but also external community and blog resources that cover more advanced topics relevant for the framework.
 ---
 
-{{< note title="Hello, there!" theme="info" >}}
-
-{{% description %}}
+{{< note theme="info" >}}
 
 Before you proceed, be sure to checkout the [{{% vendor/name %}} demo app](https://console.upsun.com/projects/create-project) and the main [Getting started guide](/get-started/here/_index.md). These two resources provide all of the core concepts and common commands you'll need to know before using the materials below.
 
 {{< /note >}}
 
-## Getting started
+Now that you have completed the [Getting started guide](/get-started/here/_index.md), there are **six remaining changes**
+required in order to have a successful deployment of Django on Upsun. Alternatively, you can read the blog post
+[_Up(sun) and running with Django_](https://upsun.com/blog/setting-up-django-on-upsun/) which walks through the steps
+required to run Django on Upsun.
+
+## `.environment`
+Django requires a few environment variables that have not been set. Open the `.environment` file and add the following:
```

Comment:
```suggestion
Your Django app, depending on your `settings.py`, may require environment variables that have not been set. Open the `.environment` file and add them:
```

Reply:
Not all Django apps will allow for this override AFAIK - pulling them in has to be done explicitly in the `settings.py` file.